tools/src/main/scala/caliban/tools/compiletime/Utils.scala

```diff
@@ -0,0 +1,17 @@
+package caliban.tools.compiletime
+
+import java.io.File
+import java.nio.file.{ FileAlreadyExistsException, Files }
+
+private[compiletime] object Utils {
+
+  def packagePath(packageName: String): String = packageName.replaceAll("\\.", "/")
```

Comment:
My gut is saying `"/"` -> `java.io.File.separator` but I've never _explicitly_ tried to support other platforms, so I don't know if internally to the JVM `"/"` gets translated to the platform-native path separator

Reply:
Fixed. Thanks!